src/content/docs/setup/configuration/commerce-configuration.mdx

```diff
@@ -51,89 +58,320 @@ The `getHeaders` function is a helper function which takes a storefront scope (l
+  - **`Magento-Store-View-Code`** - Store view for Catalog Service requests. See [Store views](https://experienceleague.adobe.com/en/docs/commerce-admin/stores-sales/site-store/store-views) for details.
+  - **`Magento-Website-Code`** - Website to connect to. See [Create websites](https://experienceleague.adobe.com/en/docs/commerce-operations/configuration-guide/multi-sites/ms-admin#step-2-create-websites) for details.
+
+</TabItem>
+
+<TabItem label="Adobe Commerce Optimizer (ACO)">
+
+Adobe Commerce Optimizer uses different header naming conventions and requires Commerce Optimizer-specific headers as defined in the [Merchandising Services API documentation](https://developer.adobe.com/commerce/services/optimizer/merchandising-services/using-the-api/#headers).
+
+```json
+{
+  "headers": {
+    "all": {
+      "Store": "{{STORE_VIEW_CODE}}"
+    },
+    "cs": {
+      "AC-Environment-Id": "{{TENANT_ID}}",
+      "AC-View-ID": "{{CATALOG_VIEW_ID}}",
+      "AC-Source-Locale": "{{LOCALE}}",
+      "AC-Price-Book-ID": "{{PRICE_BOOK_ID}}",
+      "AC-Policy-{{ATTRIBUTE_CODE}}": "{{ATTRIBUTE_VALUE}}"
+    }
+  }
+}
+```
+
+**Header Scopes:**
 
-This boilerplate `config.json` for Adobe Commerce requires correct values for your Commerce environment.
+- **`all`** - Headers applied to all GraphQL requests
+  - **`Store`** - Store view code for Core GraphQL requests. See [GraphQL Headers](https://developer.adobe.com/commerce/webapi/graphql/usage/headers/) for details.
 
-```json title="config.json" showLineNumbers collapse={20-32} ins={'1':4} ins={'2':5} ins={'3':8} ins={'4':11} ins={'5':12} ins={'6':13} ins={'7':14} ins={'8':15} ins={'9':16}
+- **`cs`** - Headers applied to all GraphQL requests
+  - **`AC-Environment-Id`** - (Required) The instance ID for the Adobe Commerce Optimizer instance. This is the tenant ID from your Commerce Optimizer instance.
+  - **`AC-View-ID`** - (Required) The unique ID assigned to the catalog view that products will be sold through. You can find this in the [Adobe Commerce Optimizer UI](https://experienceleague.adobe.com/en/docs/commerce/optimizer/setup/catalog-view).
+  - **`AC-Source-Locale`** - (Required) Catalog source locale (language or geography) to filter products for display, for example `en_US`. See the [catalog view configuration](https://experienceleague.adobe.com/en/docs/commerce/optimizer/setup/catalog-view) to determine the correct value.
+  - **`AC-Price-Book-ID`** - (Optional) Defines how prices are calculated for a specific catalog view. If not included, Merchandising Services provides a default price book with currency in US dollars.
+  - **`AC-Policy-{{ATTRIBUTE_CODE}}`** - (Optional) Policy trigger for filtering products by attribute. Examples include `AC-Policy-Brand:Cruz` for brand filtering. You can specify multiple policy headers per request.
+
+</TabItem>
+
+</Tabs>
+
+### Analytics
+
+The analytics section contains store and environment metadata for tracking and reporting.
```

Comment:
These values don't make sense for ACO storefronts -- seems like we should say something about that here.

Reply:
Are these used for Snowplow eventing? If so, then they are still required for ACO.

Reply:
I just pushed a change which refactors the analytics section. Now there are two tabs - one for ACCS/PaaS, one for ACO. Please check the ACO section. I have already modified the example and the definition to explain the hardcoded value. I could use some help with the other values, and giving reference to _how_ a user would get them.